clients/src/main/java/org/apache/kafka/common/config/TopicConfig.java

```diff
@@ -65,7 +65,9 @@
     public static final String RETENTION_BYTES_CONFIG = "retention.bytes";
     public static final String RETENTION_BYTES_DOC = "This configuration controls the maximum size a log can grow " +
```

Comment:
Maybe we should say `a partition (which consists of log segments) can grow to...`